### PR TITLE
changes made for czentrix

### PR DESCRIPTION
**Pull request by Vidyaaa24**

### CODE REVIEW

File: src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java [hunk 1]
```diff
@@ -78,8 +78,8 @@ public class MctsOutboundCallDetail {
 	private String outboundCallType;
 
 	@Expose
-	@Column(name = "CallID")  //CzentrixCallID
-	private String czentrixCallID;
+	@Column(name = "CallID")  //CallCentreID
+	private String callCentreID;
```
Comment (harimohanr) [inline src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java]: @Vidyaaa24 This property naming is wrong. It is not callCentreID. It should be callId

File: src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java [hunk 2]
```diff
@@ -193,12 +193,12 @@ public MctsOutboundCallDetail() {
 	/**
 	 * Overloaded parameter Constructor
 	 */
-	public MctsOutboundCallDetail(Long callDetailID, String outboundCallType, String czentrixCallID,
+	public MctsOutboundCallDetail(Long callDetailID, String outboundCallType, String callCentreID,
```
Comment (harimohanr) [inline src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java]: @Vidyaaa24 It should be callId

File: src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java [hunk 3]
```diff
@@ -193,12 +193,12 @@ public MctsOutboundCallDetail() {
...
 			String changeLog, String remark, String smsAdvice, java.util.Date createdDate, String createdBy) {
 
 		this.callDetailID = callDetailID;
 		this.outboundCallType = outboundCallType;
-		this.czentrixCallID = czentrixCallID;
+		this.callCentreID = callCentreID;
```
Comment (harimohanr) [inline src/main/java/com/iemr/mcts/data/agent/MctsOutboundCallDetail.java]: @Vidyaaa24 it should be callId

**Review by harimohanr — CHANGES_REQUESTED** [review]
@Vidyaaa24 Please change the callCentreId to callId

**Review by Anjana2310 — CHANGES_REQUESTED** [review]
@Vidyaaa24 please modify callCentreID  to callId in all its references

**Review by Anjana2310 — APPROVED** [review]
@Vidyaaa24 Looks good to me

**Review by harimohanr — APPROVED** [review]
@Vidyaaa24 Looks okay to me

**Pull request merged**